tests/unit/verifiers/test_neutron_gateway.py

```diff
@@ -26,10 +26,14 @@
 
 from juju_verify.verifiers import NeutronGateway, Result, Severity
 
+MIN_JUJU_VERSION = "2.8.10"
```

Comment:
It is needed?

Reply:
Nope, I guess I intended to use it in tests and then I didn't. I removed it since I was fixing merge conflict anyway.

Reply:
It is wired that `pylint` or` flake8` say nada.